File: data/src/main/java/dev/olog/data/api/deezer/DeezerArtistResponse.java
package dev.olog.data.api.deezer;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class DeezerArtistResponse {

    public List<Album> data = new ArrayList<>();

    public static class Album {

        public String picture;

        @SerializedName("picture_big")
        public String pictureBig;

        @SerializedName("picture_medium")
        public String pictureMedium;

        @SerializedName("picture_small")
        public String pictureSmall;

        @SerializedName("picture_xl")
        public String pictureXl;

    }

}

File: data/src/main/java/dev/olog/data/api/deezer/DeezerArtistResponse.kt
package dev.olog.data.api.deezer

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class DeezerArtistResponse(
    val data: List<Album>? = null,
) {

    @Serializable
    class Album(
        val picture: String? = null,
        @SerialName("picture_big")
        val pictureBig: String? = null,
        @SerialName("picture_medium")
        val pictureMedium: String? = null,
        @SerialName("picture_small")
        val pictureSmall: String? = null,
        @SerialName("picture_xl")
        val pictureXl: String? = null,
    )
}